bin/release_notes.sc

```diff
@@ -0,0 +1,114 @@
+// 2.12.12
```

Comment:
So it depends if this for sure needs 2.12.12 or not, but with the instructions down below via `cs install ammonite`, they more than likely will get the 2.13 version of Ammonite, not 2.12. If 2.12 is a hard requirement in the instructions we might want to show an alternative with just using `cs launch <specific ammonite>`. Or else this could cause weird issues.

Reply:
I will just remove it, it was leftover from some issues before :sweat_smile: